net/udpspeeder/Makefile

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright (c) 2017 Yu Wang <wangyucn@gmail.com>
+# Copyright (c) 2020 Nishant Sharma <nishant@hopbox.in>
+#
+# This is free software, licensed under MIT
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=udpspeeder
+PKG_VERSION:=20200818.0
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
```

Comment:
Upstream has tagged releases. Please use those instead of git.

git grep the repo for "codeload" to see examples.

Reply:
Removed proto git and instead using codeload. Thanks again.